tasks/create-internal-request/create-internal-request.yaml

```diff
@@ -18,20 +18,20 @@ spec:
     - name: request
       type: string
       description: request type
-    - name: params
+    - name: internalRequestParams
       type: string
       description: Internal Request parameters
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
```

Comment:
Seems the workspace is called `input` in this task. Can we update the description to use the proper name?